Translation/en/RedirectedResources/assets/abdata/h/list/01/personality_voice_c07/translation.txt

```diff
@@ -2428,16 +2428,16 @@
 ちんちん気持ちよくて……おかしくなりそうだ=Your cock feels so good... I'm gonna go crazy.
 聞こえるか？　ぐちょぐちょって、オマンコ喜んでるぞ=Can you hear it? The sopping sounds my pussy is making. It's because it's really happy.
 顔、見ないでくれっ、絶対変になってるから……=Please don't look at my face--, I've definitely got a weird expression right now...
-奥のいいとこ、カリでこすられてるっ=You're rubbing my sensitive spot, with your dick's head--
-すごいな……ちんちんずぶずぶ入ってるぞ=Amazing... his dick is going in so deep.
+奥のいいとこ、カリでこすられてるっ=You're rubbing my sensitive spot, with your cock's head--
+すごいな……ちんちんずぶずぶ入ってるぞ=Amazing... his cock is going in so deep.
 ああ……私もすごく感じてるぞ……=Yeah... I feel really good too...
 そんなに気持ちいいのか？　声いっぱい出てるぞ=Does it feel that good? You're moaning a lot.
 んくっ、こっちもやばい……感じる……=Nnkuh--, I'm going crazy too... this feels great...
 そろそろイキそうか？　顔が真っ赤だぞ……=Are you about to cum? Your face is real flushed...
 ふふ、思いっきりイキたいって顔してるな=Fufu, your expression is screaming volumes about just how hard you wanna cum.
 私ももうヤバい……耐えらんないかも……=I'm also nearly there... I might not be able to bear it...
 このまま二人ともイカせてもらおうぜ……=We'll have you make the two of us cum like this...
-もう限界か……？　オチンポ気持ちいいもんな=At your limit...? His dick really does feel good, afterall.
+もう限界か……？　オチンポ気持ちいいもんな=At your limit...? His cock really does feel good, afterall.
```

Comment:
Space in "after all".

Reply:
Most translations use afterall without space. It can be fixed separately with a single find and replace.